account_payment_order_notification/views/account_payment_order_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_payment_order_form" model="ir.ui.view">
+        <field name="name">account.payment.order.form</field>
+        <field name="model">account.payment.order</field>
+        <field
+            name="inherit_id"
+            ref="account_payment_order.account_payment_order_form"
+        />
+        <field name="arch" type="xml">
+            <button name="action_cancel" position="before">
+                <button
+                    type="action"
+                    name="%(wizard_account_payment_order_notification_action)d"
+                    string="Send mails"
+                    attrs="{'invisible': [('state', 'in', ('draft','cancel'))]}"
```

Comment:
If we want to be inline with the text of the mail, this button should be only available in the states `uploaded` and `done`.